src/main/java/com/cburch/logisim/std/memory/TFlipFlop.java

```diff
@@ -36,10 +35,10 @@ public TFFHDLGeneratorFactory() {
     }
 
     @Override
-    public ArrayList<String> GetUpdateLogic() {
-      return LineBuffer.getBuffer()
-          .add("{{1}} s_next_state {{2}} s_current_state_reg {{3}} T;", Hdl.assignPreamble(), Hdl.assignOperator(), Hdl.xorOperator())
-          .getWithIndent();
+    public LineBuffer getUpdateLogic() {
+      return LineBuffer.getHdlBuffer()
+          .pair("xor", Hdl.xorOperator())
```

Comment:
Maybe add that pair to LineBuffer's `addHdlPairs()` directly?

Reply:
I agree, this one is missing

Reply:
Added